graph.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+class Graph:
+    def __init__(self) -> None:
+        self.adjacency_list = defaultdict(list)
+    
+    def add_edge(self, u, v):
+        self.adjacency_list[u].append(v)
+        self.adjacency_list[v].append(u)
+        
+    def dfs(self, source):
```

Comment:
Method name is 'dfs' ie 'depth first search' where is the searching parameter? Is this search or traversal?

Reply:
it should be dfs_travesral, I'll change